.circleci/config.yml

```diff
@@ -38,18 +38,19 @@ jobs:
             ensure-version-bumped.sh
             prohibit-tab-character.sh
             prohibit-trailing-whitespace.sh
+
       - run:
           name: Run tests against Python 3.6 and git 1.8.0 (without checking coverage)
-          command: PYTHON_VERSION=3.6 GIT_VERSION=1.8.0  bash ci/tox/ci-run.sh  # Earliest versions of python/git supported by git-machete
+          command: PYTHON_VERSION=3.6 GIT_VERSION=1.8.0  bash ci/ci-run.sh tox # Earliest versions of python/git supported by git-machete
```

Comment:
I'd consider renaming `ci-run.sh` to sth more specific now that its location changed... maybe `run-dockerized.sh`?